packages/fuselage-toastbar/src/ToastBarZone.tsx

```diff
@@ -4,10 +4,22 @@ import type { ReactNode, ReactElement } from 'react';
 import type { ToastBarPayload } from './ToastBarContext';
 
 const positionProps = {
-  'top-start': 'top: 0; left: 0;',
-  'top-end': 'top: 0; right: 0;',
-  'bottom-start': 'bottom: 0; left: 0;',
-  'bottom-end': 'bottom: 0; right: 0;',
+  'top-start': {
+    ltr: 'top: 0; left: 0; right: unset; bottom: unset;',
+    rtl: 'top: 0; right: 0; left: unset; bottom: unset;',
+  },
+  'top-end': {
+    ltr: 'top: 0; right: 0; left: unset; bottom: unset;',
+    rtl: 'top: 0; left: 0; right: unset; bottom: unset;',
+  },
+  'bottom-start': {
+    ltr: 'bottom: 0; left: 0; right: unset; top: unset;',
+    rtl: 'bottom: 0; right: 0; left: unset; top: unset;',
+  },
+  'bottom-end': {
+    ltr: 'bottom: 0; right: 0; left: unset; top: unset;',
+    rtl: 'bottom: 0; left: 0; right: unset; top: unset;',
+  },
```

Comment:
`question` Why not using logical properties if the browser's CSS API supports it?

Reply:
due the unit tests limitation